src/python/WMCore/ReqMgr/Web/utils.py

```diff
@@ -264,3 +271,18 @@ def sort(docs, sortby):
     "Sort given documents by sortby attribute"
     for doc in docs:
         yield doc
+        
+def reorder_list(org_list, first):
+    """
+    if the first is in the list.
+    move the first in front of the list
+    if not, add first to the list
+    """
+    new_list = list(org_list)
+    try:
+        i = new_list.index(first)
+        new_list[0], new_list[i] = new_list[i],  new_list[0]
```

Comment:
The code does is swapping the items on the list. So it will have correct default selected options in the html options. (Code itself works, I tested).
I know that condition won't be ever happen except the case you describe or if we add remove certain options between the deployment and acdc is created from the previous workflow. However, I wasn't sure whether remove parameter still need to be set for previous workflow. In that case we can run the acdc at all using GUI interface. The chances would be close to none but I thought would be safer to do.